website/source/docs/providers/aws/r/opsworks_instance.html.markdown

```diff
@@ -0,0 +1,133 @@
+---
+layout: "aws"
+page_title: "AWS: aws_opsworks_instance"
+sidebar_current: "docs-aws-resource-opsworks-instance"
+description: |-
+  Provides an OpsWorks instance resource.
+---
+
+# aws\_opsworks\_instance
+
+Provides an OpsWorks instance resource.
+
+## Example Usage
+
+```
+resource "aws_opsworks_instance" "my-instance" {
+  stack_id = "${aws_opsworks_stack.my-stack.id}"
+  layer_ids = [
+    "${aws_opsworks_custom_layer.my-layer.id}",
+  ]
+  instance_type = "t2.micro"
+  os = "Amazon Linux 2015.09"
+  state = "stopped"
```

Comment:
Style nit - can you format this like `hclfmt` would? (Basically align equals for adjacent attributes.)